MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp

```diff
@@ -116,7 +116,11 @@ void EnggDiffractionViewQtGUI::initLayout() {
   doSetupTabSettings();
 
   m_presenter->notify(IEnggDiffractionPresenter::Start);
-  m_presenter->notify(IEnggDiffractionPresenter::RBNumberChange);
+  // Never do this here: if the RB-number checks show a pop-up (splash
+  // message), it will be shown very early (before the interface
+  // window itself is shown) and that will cause a crash in Qt code on
+  // some platforms (windows 10, and 7 sometimes).
+  // m_presenter->notify(IEnggDiffractionPresenter::RBNumberChange);
```

Comment:
Can we remove the commented out code and update the comment to say that we delay notifying about the RB number change until the window is show?